### ISSUE TITLE
When running from systemd the syncdir option is ignored

### ISSUE THREAD

**Issue by owenhollands** [abraunegg/onedrive#160]
### Bug Report Details ###
**Describe the bug**
When running from systemd the syncdir option is ignored and the directory used is always /rootOneDrive even if the directory is specified in ~/.config/onedirve/config or wiht the --sydndir option

**Application and Operating System Details:**
- OS: Output of `uname -a` Linux *****.*****.id.au 3.10.0-862.9.1.el7.x86_64 #1 SMP Mon Jul 16 16:29:36 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
- Application version: Output of `onedrive --version` onedrive v2.1.2
- OneDrive Account Type - individial 
- DMD or LDC compiler version `dmd --version` or `ldmd2 --version` DMD64 D Compiler v2.082.0

**To Reproduce**
Steps to reproduce the behavior if not causing an application crash:
start the onedirve deamon using the default systemd scripts.  The sync directory will be /rootOnedrive when the root home directory is /root)

If issue is replicated by a specific 'file' or 'path' please archive the file and path tree & email to support@mynas.com.au 

**Complete Verbose Log Output**
A clear and full log of the problem when running the application in the following manner (ie, not in monitor mode):
```
onedrive --synchronize --verbose <any of your other needed options>
```

Run the application in a separate terminal window or SSH session and provide the entire application output including the error & crash. When posing the logs, Please format log output to make it easier to read. See https://guides.github.com/features/mastering-markdown/ for more details.

Application Log Output:
```
Verbose console log output goes here
```

**Screenshots**

**Additional context**

### Bug Report Checklist ###
- [ ] Detailed description
- [ ] Reproduction steps (if applicable)
- [ ] Verbose Log Output

**Comment by abraunegg** [comment]
Can you provide some more verbose debug logs for this issue?

I came across this issue in testing (#131) and fixed via PR #132 - so it would be interesting to understand how and where this is being broken, as this should have been resolved.

**Comment by abraunegg** [comment]
> The sync directory will be /rootOnedrive when the root home directory is /root

This is the same issue as #169, fixed in PR #170
